AsyncDisplayKit/ASControlNode.mm

```diff
@@ -86,9 +87,11 @@ - (id)init
 
   // As we have no targets yet, we start off with user interaction off. When a target is added, it'll get turned back on.
   self.userInteractionEnabled = NO;
+  self.isAccessibilityElement = YES;
```

Comment:
Does the accessibility system automatically consider views with gesture recognizes to be accessibility elements? If not, we may want to add support for this in _ASDisplayView, where elements with a nonzero number of gesture recognizers are marked as being accessibility elements.

More importantly though, I think we need to add this line only in the addTarget: method.  ASTextNode, ASImageNode, ASMapNode, etc are all ASControlNodes for convenience, and yet many of them are not actually interactive elements. This explains the userInteractionEnabled = NO above as well.